draft-ietf-rtcweb-jsep.xml

```diff
@@ -1095,6 +1095,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           described in
           <xref target="sec.addTransceiver" />.</t>
         </section>
+        <section title="removeTrack" anchor="sec.removeTrack">
+
+          <t>The removeTrack method removes a MediaStreamTrack from the
+          PeerConnection, using the RTCRTpSender argument to indicate
+          which track is removed. The sender's track attribute is set to
+          null, and the sender stops sending.  Future calls to createOffer
+          will mark the media description corresponding to the transceiver
```

Comment:
since we are talking about the sender, should this say "media description associated with the sender"?